concent_api/verifier/tasks.py

```diff
@@ -40,6 +41,12 @@
 logger = logging.getLogger(__name__)
 
 
+def check_if_sizes_in_pixels_are_equal(image1: ndarray, image2: ndarray) -> bool:
```

Comment:
the name of the function does not follow convetion of `bool` returning functions. Please rename it to sth like: `are_pixel_sizes_equal` or `are_image_sizes_equal` or `are_images_resolutioins_equal`

Reply:
done

Reply:
Move this function to `verifier/utils.py`.

Reply:
Yeah. If anyone ever wonders why we have a rule against function names starting with `check_` - that's why ;-)

Reply:
yes... done- renamed and moved :)